src/components/navbar.js

```diff
@@ -8,25 +8,31 @@ const Navbar = () => {
     {
       id: 1,
       path: '/',
-      text: 'Books',
+      text: 'BOOKS',
+      color: '#121212',
     },
     {
       id: 2,
       path: '/categories',
-      text: 'Categories',
+      text: 'CATEGORIES',
+      color: '#121212',
     },
-
   ];
   return (
     <nav className="navBar" style={navStyle}>
-      <h1> Bookstore CMS </h1>
-      <ul style={{ ...navStyle, listStyle: 'none', columnGap: '2px' }}>
+      <ul
+        className="navList"
+        style={{ ...navStyle, listStyle: 'none', columnGap: '2px' }}
+      >
+        <h1> Bookstore CMS </h1>
```

Comment:
- [ ] The navigation bar is different than the original. To stick to the design kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/174399386-0d63930f-2ec1-4641-8c5e-49de1c5d37b3.png)

___
ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/174399427-1a00882f-35cb-45f7-8e04-8643d15d085a.png)